doc/connectivity/bluetooth/api/mesh/shell.rst

```diff
@@ -269,67 +269,67 @@ To allow a device to provision devices, the :kconfig:option:`CONFIG_BT_MESH_PROV
 
 To allow a device to provision devices over GATT, the :kconfig:option:`CONFIG_BT_MESH_PROVISIONER` and :kconfig:option:`CONFIG_BT_MESH_PB_GATT_CLIENT` configuration options must be enabled.
 
-``mesh prov remote-gatt <UUID> <NetKeyIndex> <addr> <AttentionDuration>``
--------------------------------------------------------------------------
+``mesh prov remote-gatt <UUID> <NetIdx> <Addr> <AttentionDuration(s)>``
+-----------------------------------------------------------------------
 
-	Provision a nearby device into the mesh. The mesh node starts scanning for connectable advertising for PB-GATT with the given UUID. Once found, the unprovisioned device will be added to the mesh network with the given unicast address, and given the network key indicated by ``NetKeyIndex``.
+	Provision a nearby device into the mesh. The mesh node starts scanning for connectable advertising for PB-GATT with the given UUID. Once found, the unprovisioned device will be added to the mesh network with the given unicast address, and given the network key indicated by ``NetIdx``.
 
 	* ``UUID``: UUID of the unprovisioned device.
-	* ``NetKeyIndex``: Index of the network key to pass to the device.
-	* ``addr``: First unicast address to assign to the unprovisioned device. The device will occupy as many addresses as it has elements, and all must be available.
+	* ``NetIdx``: Index of the network key to pass to the device.
+	* ``Addr``: First unicast address to assign to the unprovisioned device. The device will occupy as many addresses as it has elements, and all must be available.
 	* ``AttentionDuration``: The duration in seconds the unprovisioned device will identify itself for, if supported. See :ref:`bluetooth_mesh_models_health_srv_attention` for details.
 
-``mesh prov uuid [UUID: 1-16 hex values]``
-------------------------------------------
+``mesh prov uuid [UUID(1-16 hex)]``
```

Comment:
Have to specify whether it is aligned from right or left. There was already such question from user.